### PR TITLE
Notify user via email when they're added to a project

### LINKED ISSUE #306: Notify user via email when they're added to a project

### PR DESCRIPTION
**Pull request by elias-ba**
## Related issue

Fixes #306 

## Checklist before requesting a review

- [x] I have performed a **self-review** of my code
- [x] I have verified that all appropriate **authorization policies** have been implemented and tested
- [x] If needed, I have updated the **changelog**
- [x] Amber has **QA'd** this feature

### CODE REVIEW

**Review by amberrignell — CHANGES_REQUESTED** [review]
Just some small language changes for the email subject and body, and a request to link to the project. Thanks Elias

File: lib/lightning/accounts/user_notifier.ex [hunk 1]
```diff
@@ -40,6 +41,20 @@ defmodule Lightning.Accounts.UserNotifier do
...
+  """
+  def deliver_project_addition_notification(user, project) do
+    role = Projects.get_project_user_role(user, project)
+
+    deliver(user.email, "Project Addition", """
+
+    Hi #{user.first_name},
+
+    You've just been added to the project #{project.name} as #{role}
+    """)
```
[the remark this answers is not in the document]
Comment (elias-ba) [reply lib/lightning/accounts/user_notifier.ex:60]: Done. Thanks

**Review by amberrignell — APPROVED** [review]
Nice one, thanks Elias. @taylordowns2000 if you are happy with the code you can go ahead and merge.

**Review by taylordowns2000 — APPROVED** [review]
@elias-ba , @amberrignell , i've made a change to use the correct indefinite article. code looks good. thank you!

**Pull request merged**